readme/169-qr-code-specification-1.1.0.md

```
@@ -0,0 +1,377 @@
+    00000",
+    "dataFormat": "image",
+    "dataSubFormat": "png"
+  }
+}
+```
+
+* **Convert to Claim 169 Format**
+  * Transform the JSON data into the required Claim 169 structure.
+  * Refer to the sample converted data for guidance.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)